app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -160,21 +160,45 @@ public void onTimeUpdated(int newmins) {
                     ucalendar.add(Calendar.HOUR_OF_DAY, -1); // hack for warmup time
                 }
 
-                realStartSensor();
+                startSensorOrAskForG6Code();
             }
         });
         timePickerFragment.show(activity.getFragmentManager(), "TimePicker");
     }
 
-    private void realStartSensor() {
+    private void startSensorOrAskForG6Code() {
```

Comment:
Isn't `realStartSensor()` also used by other sensors like Libre?

Reply:
This is just a method rename -- clarified below